setup.sh

```diff
@@ -288,33 +187,29 @@ function _main
   fi
 
   local OPTIND
-  while getopts ":c:i:p:hzZ" OPT
+  while getopts ":c:i:p:zZ" OPT
   do
     case ${OPT} in
-      i ) IMAGE_NAME="${OPTARG}" ;;
-      z ) USE_SELINUX=":z"     ;;
-      Z ) USE_SELINUX=":Z"     ;;
-      c )
-        # container specified, connect to running instance
-        CONTAINER_NAME="${OPTARG}"
-        ;;
+      ( i ) IMAGE_NAME="${OPTARG}"       ;;
+      ( z | Z ) USE_SELINUX=":${OPTARG}" ;;
+      ( c ) CONTAINER_NAME="${OPTARG}"   ;;
```

Comment:
@georglauterbach

Just noticed this: Why are there all these '(' added in the case statement?

Reply:
It seems like no one knows this syntax :D I have seen it a little while ago by accident and my brace highlighting extensions liked it too. What do you think of it? I personally like the braces on both sidesy

Reply:
I'm ok with it as long as it's not a requirement for the rest of us. I've never seen or used it before myself, so I'd like to not have to remember and change my existing habits :)

Reply:
TIL that this is [valid syntax](https://www.gnu.org/software/bash/manual/bash.html#index-case) 🎉
IMO we should stick to well-known syntax patterns, as this will probably confuse everyone on a first look.

On the other hand, we can see that as a security feature. Bad guys analyzing our scripts to find vulnerabilities will immediately think: "WTF is this. I probably should not mess with guys that use such a syntax.". 😆

Reply:
> "WTF is this. I better should not mess with guys that use such a syntax."

:joy: 

> What do you think of it?

I like it. Shell syntax has always been weird looking to me though with stuff like `case esac` and `if then fi`, but I prefer parenthesis on both ends to wrap a case condition, little less foreign looking.

Reply:
I'm fine with everyone using his/her own style in this special instance:) We're not linting this, and I and @polarathene as well as @casperklein and @NorseGaud can use their preferred syntax too :D

Reply:
Yeah. What confuse me, in BASH `()` means subshell. So for me it feels not consistent, using () for other things. Apart from that, `()` looks nicer.